### PR TITLE
add a cron to send project expiration mail to owner

### PR DESCRIPTION
**Pull request by FanchTheSystem**
copy of the test on user (with the 60/15/3 day counter) but for project owner

### CODE REVIEW

File: cron/test_projects.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            do_notify = true;
+        } else if (project.expiration_notif == 2 && (project.expire <= in3Days)) {
+            project.expiration_notif = 3;
+            do_notify = true;
+        }
+        if (!do_notify) {
+            continue;
+        }
+        console.log(`Project will expire, send notication number ${project.expiration_notif} to ${project.id}`);
+        let user = await dbsrv.mongo_users().findOne({uid: project.owner});
```
Comment (osallou) [inline cron/test_projects.js:67]: should manage case where user is not found

**Review by osallou — COMMENTED** [review]
there is also a cron/test_quote_expire.js, badly named as it does not check quotas, but project expiration. Maybe it should just be replaced with this code (and call in cron/gomngr.sh)

File: cron/test_projects.js (hunk 1 above)
Comment (FanchTheSystem) [reply cron/test_projects.js]: 701f4411fa1e46e0b91a3983e1a3048bda268613 Done, and I also add a check on project owner

**Comment by FanchTheSystem** [comment]
> there is also a cron/test_quote_expire.js, badly named as it does not check quotas, but project expiration. Maybe it should just be replaced with this code (and call in cron/gomngr.sh)

Can't find test_quote_expire.js in the repo

**Comment by osallou** [comment]
cron/project_quota_expire.js , typo error...

**Comment by osallou** [comment]
should call this script in cron/gomngr.sh in this case in place of project_quota_expire

**Comment by FanchTheSystem** [comment]
I have added the new script in the cron.

About project_quota_expire I will modify it in another PR to take action on project expiration (same kind of action we already have for user expiration)

**Comment by osallou** [comment]
project_quota_expire had a wrong name, it simply test project expiration, so your script should just replace it (so delete it and replace in cron/gomngr.sh)

**Comment by FanchTheSystem** [comment]
> project_quota_expire had a wrong name, it simply test project expiration, so your script should just replace it (so delete it and replace in cron/gomngr.sh)

done

File: cron/gomngr.sh [hunk 2]
```diff
@@ -153,13 +145,21 @@ while true; do
...
-      rm /tmp/gomngr.lock
-      rm /tmp/gomngr.list
-      echo "Exit requested"
-      exit 0
+        rm /tmp/gomngr.lock
+        rm /tmp/gomngr.list
+        echo "Exit requested"
+        exit 0
+    fi
+    echo "Check for account upcoming expiration"
```
Comment (osallou) [inline cron/gomngr.sh:153]: Project not account (and will be ok for merge)

**Pull request merged**